### ISSUE TITLE
Won't install on Windows 8.1 64 bit

### ISSUE THREAD

**Issue by juranta** [seongjaelee/notational-velocity#17]
I tried to install notational-velocity plugin on Windows 8.1 64 bit Atom version 0.205.0. When I try to install it settings, it fails with the output below. The plugin works on my Mac and this is probably some native Windows issue. There a similar issue for instance here: https://github.com/izuzak/atom-pdf-view/issues/50

Maybe it's a plugin issue. I can also try to look further to this issue. I have to use Windows at work, so I want to get something like nvALT work so that I can try to repeat some of my Mac workflow.

```
Installing “notational-velocity@0.1.0” failed.Hide output…

> runas@2.0.0 install C:\Users\name\AppData\Local\Temp\apm-install-dir-11553-14428-1bzevon\node_modules\notational-velocity\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


C:\Users\name\AppData\Local\Temp\apm-install-dir-11553-14428-1bzevon\node_modules\notational-velocity\node_modules\pathwatcher\node_modules\runas>node "C:\Users\name\AppData\Local\atom\app-0.205.0\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 

gypnpm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\name\\atom\\app-0.205.0\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\name\\atom\\app-0.205.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\juhran\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\juhran\\.atom\\.apmrc" "install" "C:\\Users\\name\\Temp\\d-11553-14428-913892\\package.tgz" "--target=0.22.0" "--arch=ia32"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

**Comment by seongjaelee** [comment]
Seems like there is an issue with pathwatcher. For pdf-view, it could switch to atom's internal functionality, but in our case, watch method (that monitors file/directory change) is not supported from atom library, so a simple change as in is not available. I'll investigate more later.

**Comment by deltaidea** [comment]
Can it be the case that it just can't find Visual Studio?

**Comment by seongjaelee** [comment]
It should be fixed in 0.2.0 with #24. If the problem still appears, please reopen this issue.
